actionview/test/activerecord/partial_rendering_query_test.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "active_record_unit"
+
+class PartialRenderingQueryTest < ActiveRecordTestCase
+  def setup
+    @view = ActionView::Base
+      .with_empty_template_cache
+      .with_view_paths(ActionController::Base.view_paths, {})
+
+    @queries = []
+
+    ActiveSupport::Notifications.subscribe("sql.active_record") do |*, payload|
```

Comment:
Looking at some of the other tests, you'll want to store the subscription in a variable, so you can unsubscribe at the end of the test. Right now, all subsequent tests are pushing their SQL into this array (a memory leak in the test suite).

For example: https://github.com/rails/rails/blob/f130c780f6c6cb09ad68ac89de9af90102fa2d80/activerecord/test/cases/instrumentation_test.rb#L22

Reply:
@natematykiewicz Thanks, I've done as you suggested.